config/build.xml

```diff
@@ -40,6 +40,8 @@
       <property key="checkstyle.importcontroltest.file" file="config/import-control-test.xml"/>
       <property key="checkstyle.suppressions.file"
                 file="config/suppressions.xml"/>
+      <property key="checkstyle.suppressions-xpath.file"
+                file="config/suppressions-xpath.xml"/>
```

Comment:
this file is not used in project, #4877 , sorry that we did not find time for cleanup.
please remove this update.

Reply:
done